pallets/subtensor/src/errors.rs

```diff
@@ -50,67 +62,65 @@ mod errors {
         InvalidSeal,
         /// the dispatch attempts to set weights on chain with where any normalized weight is more than MaxWeightLimit.
```

Comment:
The dispatch is attempting to set weights on chain with weight value exceeding the MaxWeightLimit (max_weight_limit subnet hyperparameter).